### ISSUE TITLE
Blacklisted

### ISSUE THREAD

**Issue by wbyoung** [deepak1556/gulp-browserify#64]
Just wanted to share that this plugin [has been blacklisted](https://github.com/gulpjs/plugins/issues/33) by gulp.js suggesting to use the browserify module directly instead. I haven't dug into what it would mean to just use _browserify_. If it'd be difficult, perhaps you should open a pull request to take this off the blacklist.

**Comment by battlesnake** [comment]
@mshick Agreed!  My build system shouldn't die on syntax errors, it should just report the error and continue watching!
